lib/mbta.js

```diff
@@ -1,12 +1,140 @@
-'use strict'
+//Conor Jennings - MBTA Assignment
+let redStops = ['South Station', 'Park Street', 'Kendall', 'Central', 'Harvard', 'Porter', 'Davis', 'Alewife']
+let greenStops = ['Government Center', 'Park Street', 'Boylston', 'Arlington', 'Copley', 'Hynes', 'Kenmore']
+let orangeStops = ['North Station', 'Haymarket', 'Park Street', 'State', 'Downtown Crossing', 'Chinatown', 'Back Bay', 'Forest Hills']
+let totalNumberStops
 
-// Code here.
 
-const stopsBetweenStations = (startLine, startStation, endLine, endStation) => {
+const stopsBetweenStations = function(startLine, startStation, endLine, endStation) {
 
-}
+	if (startLine === endLine) {
+		changeTrains = false
+	} else {
+		changeTrains = true
+	}
 
-module.exports = {
-  stopsBetweenStations,
-  stretch: false
-}
+
+	//Didn't change trains:
```

Comment:
Your logic is sound here and this works, so nice job. However, it would be a lot less work if you had a function that could calculate the number of stops traveled no matter what line is chosen. Using the object example from above, you could do something like this: ` let firstStop = subwayLines[startline][startStation]...etc`